projects/storefrontapp/src/environments/environment.ts

```diff
@@ -9,7 +9,7 @@ export const environment: Environment = {
   production: false,
   occBaseUrl:
     build.process.env.SPARTACUS_BASE_URL ??
-    'https://spartacus-dev0.eastus.cloudapp.azure.com:9002',
+    'https://spartacus-dev2.eastus.cloudapp.azure.com:9002',
```

Comment:
Reminder for myself to remove it.
Please ignore this comment until the final merge.